tiatoolbox/utils/visualization.py

```diff
@@ -68,7 +68,7 @@ def overlay_patch_prediction(
             label_info[label_uid] = (str(label_uid), random_colour)
     else:
         # may need better error message
```

Comment:
May need a better error message for what? I am a little confused by this comment

Reply:
https://github.com/TIA-Lab/tiatoolbox/blob/1520a389f88bc6f0b71d7f48c0e7e4446d7d97bc/tiatoolbox/utils/visualization.py#L76
https://github.com/TIA-Lab/tiatoolbox/blob/1520a389f88bc6f0b71d7f48c0e7e4446d7d97bc/tiatoolbox/utils/visualization.py#L79

For example this, will only tell sthg wrong but is not informative enough to directly point out why, making L76 and L79 very uninformative.